test/karma/launchers.js

```diff
@@ -211,77 +211,75 @@ module.exports = {
 		os_version: "High Sierra"
 	},
 
-	"bs_ios-5.1": {
-		base: "BrowserStack",
-		device: "iPhone 4S",
-		os: "ios",
-		os_version: "5.1"
-	},
-	"bs_ios-6.0": {
-		base: "BrowserStack",
-		device: "iPhone 5",
-		os: "ios",
-		os_version: "6.0"
-	},
-	"bs_ios-7.0": {
+	"bs_ios-10.3": {
 		base: "BrowserStack",
-		device: "iPhone 5S",
+		device: "iPhone 7",
 		os: "ios",
-		os_version: "7.0"
+		os_version: "10.3",
+		real_mobile: true
 	},
-	"bs_ios-8.3": {
+	"bs_ios-11.0": {
```

Comment:
I think we're fine with the latest version of each iOS major, especially that the API usually deletes the older ones.